core/frontend/src/Tiles.ts

```diff
@@ -179,6 +179,20 @@ export class Tiles implements Iterable<{ supplier: TileTreeSupplier, id: any, ow
     return tree;
   }
 
+  public resetTileTreeOwner(id: any, supplier: TileTreeSupplier) {
```

Comment:
Why is this `@public`? If intentional, it should be documented. But seems dangerous.

Reply:
You are right, exposing tile tree ids is probably a bad idea. Only Viewport.resetMapLayer should be visible.

Actually even that one could be discussed... down the road I need a way to reset a maplayer once a "user fixable" error occured, for example missing/expired security token.   Adding / removing map-layer instance won't do the trick since the tile tree is re-used if the map-layer settings match with a previous layer.

Reply:
I'm not talking about the `id`, I'm more concerned about the concept of destroying tile trees willy-nilly. Do you see a general need to do such a thing by users other than the map layer widget?

Reply:
It's not really specific to the map-layers widget... the main problem is when a problem occurs during provider initialization, or a tile fetch fails for whatever reason, there is no way to recover from that.

The only way I could find is to have the tile tree re-created, and that normally only happens when a setting part of the MapTreeId change... other than that you're stuck for the entire session  with either a zombie map-layer (failed initialization) or a layer displaying low resolution tiles (because even though you can successfully load higher resolution later on, once a tile fetch has failed, it will force the display of lower resolutions (i.e. replacement tiles).

Reply:
A failure to fetch a single tile ideally does not result in discarding all of the tiles.

Can you apply an appropriate release tag please?